tutorials/assets_pipeline/importing_scenes.rst

```diff
@@ -29,6 +29,33 @@ Copy the scene file together with the textures and mesh data (if separate) to
 the project repository, then Godot will do a full import when focusing the
 editor window.
 
+3D asset direction conventions
+------------------------------
+
+Godot uses a right-handed, Y-is-up coordinate system, with the -Z axis as
+the camera's forward direction. This is the same as OpenGL. This implies
+that +Z is back, +X is right, and -X is left for a camera.
+
+The convention for 3D assets is to face the opposite direction as the camera,
+so that characters and other assets are facing the camera by default.
+This convention is extremely common in 3D modeling applications, and is
+`codified in glTF as part of the glTF 2.0 specification <https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#coordinate-system-and-units>`.
+This means that for oriented 3D assets (such as characters),
+the +Z axis is the direction of the front, so -Z is the rear,
++X is the left side, and -X is the right side for a 3D asset.
+In Blender, this means that +Y is rear and +Y is front for an asset.
```

Comment:
The asset's front in Blender is -Y and world space forward is +Y. However, it is complicated but for cameras and lights, the front is -Z even if it is Blender.

Reply:
Ah yeah that's a typo, I accidentally wrote +Y twice. Good catch.

I think we only need to talk about the asset forward to Blender since it's weird with cameras/etc.